page_shield_connections.go

```diff
@@ -0,0 +1,88 @@
+package cloudflare
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/goccy/go-json"
+)
+
+// ListPageShieldConnectionsParams represents parameters for a page shield connection request.
+type ListPageShieldConnectionsParams struct {
```

Comment:
for these params, are they expected to be URL query parameters if passed in? if so, we will need to use the `url` struct tags to handle converting them for us. examples: https://github.com/cloudflare/cloudflare-go/blob/0904ebc0820ca6aa225455154237328e34c67754/dns.go#L56-L66

Reply:
url params